### ISSUE TITLE
API 404's

### ISSUE THREAD

**Issue by nolastan** [judsonmitchell/huey#1]
I'm excited to find this – was going to build a scraper myself and this will save me a ton of time 😌

In trying out some of the samples, it appears `http://hueylaw.org/api/*` redirects to a 404 on `https://loyolalawtech.org/`.

**Comment by judsonmitchell** [comment]
Yes, we had a server crash and there are still some configuration issues with apache. I will look into it.
